### PR TITLE
VP decay

### PR DESCRIPTION
**Pull request by sprunk**

### CODE REVIEW

File: ZkLobbyServer/SpringieInterface/PlanetWarsTurnHandler.cs [hunk 1]
```diff
@@ -314,9 +314,13 @@ public static void EndTurn(string mapName, List<string> extraData, ZkDataContext
         // process faction energies
         foreach (var fac in db.Factions.Where(x => !x.IsDeleted)) fac.ProcessEnergy(gal.Turn);
 
-        // process production
+        // process production (incl. victory points)
         gal.ProcessProduction();
 
+        var VP_facs = db.Factions.Where(x => x.VictoryPoints > 0);
```
Comment (Licho1) [inline ZkLobbyServer/SpringieInterface/PlanetWarsTurnHandler.cs]: CodeStyleNazi: local variables dont start with capital and we arent using _ anywhere else

File: ZkLobbyServer/SpringieInterface/PlanetWarsTurnHandler.cs [hunk 2]
```diff
@@ -314,9 +314,13 @@ public static void EndTurn(string mapName, List<string> extraData, ZkDataContext
         // process faction energies
         foreach (var fac in db.Factions.Where(x => !x.IsDeleted)) fac.ProcessEnergy(gal.Turn);
 
-        // process production
+        // process production (incl. victory points)
         gal.ProcessProduction();
```
Comment (Licho1) [inline ZkLobbyServer/SpringieInterface/PlanetWarsTurnHandler.cs]: if this does not do DbSave next line might get outdated state (IE first tick with influence wont decay)

Comment (GoogleFrog) [reply ZkLobbyServer/SpringieInterface/PlanetWarsTurnHandler.cs]: First tick with influence should decay. Add DB save?

Comment (Anarchid) [reply ZkLobbyServer/SpringieInterface/PlanetWarsTurnHandler.cs]: https://github.com/ZeroK-RTS/Zero-K-Infrastructure/pull/2177/commits/42f35c19344305ef539faa25a70b0432d0785233

**Pull request merged**